### ISSUE TITLE
Extensions and Persona should not be in History container

### ISSUE THREAD

**Issue by sonphnt** [microsoft/azurechat#324]
I can see we are storing **Extensions** and **Personas** configuration in CosmosDB - **History** container.

Should we store them in **Config** container where we already store **Prompt** entity that will make more sense than History container?

**Comment by FlorianSchloesser** [comment]
I noticed that too.

Since we are in Europe and have to comply with GDPR - and we set up an automatic deletion which was a pain, because I could not just set a ttl, because then personas etc. would also be deleted.

After changing this in my case it got much easier. It just makes sense to move this to the config container.

**Comment by DakotaWray2** [comment]
@FlorianSchloesser Any unintended consequences you observed with that change? I also want to set TTL on my history container. Thank you

**Comment by DakotaWray2** [comment]
@FlorianSchloesser Thank you for the detailed response!

**Comment by FlorianSchloesser** [comment]
@DakotaWray2 
Youre welcome! 
One more thing I forgot to mention:
Keep in mind, that Personas and Extensions are stored in the history container (at least in the version I forked - though the Microsoft Devs did start doing some updates).

They can and should be in the Config DB - as the prompts are - so if you really set a ttl those will be deleted as well! But changing containers for them should not be an issue - as you can just call the relevant functions for the ConfigDB in the code instead - if you are already in production you might have to migrate, but nothing that cannot be done with a script ;)

**Comment by jimmylevell** [comment]
By introducing the following changes, we were able to move the extensions, persona and prompts to a dedicated config container:

- cosmos.ts: define a new config container
- within the persona, persona and extension service replace the existing HistoryContainer with the newly created ConfigContainer. 

It allowed us to use TTL within the History but not for the Config container.

**Comment by olafwrieden** [comment]
Good catch! Anyone happy to make a small PR for this issue? I'll happily help with the review.

**Comment by jimmylevell** [comment]
@olafwrieden of course! I just created a PR (https://github.com/microsoft/azurechat/pull/500) to the best of my knowledge. Curious about your thougths, as I simply changed the historyContainer to the configContainer on the related services. 
Thanks for the support!
